packages/react-ssr/CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## [0.26.0](https://github.com/cds-snc/gcds-components/compare/gcds-components-react-ssr-v0.25.1-canary.0...gcds-components-react-ssr-v0.26.0) (2024-09-12)
```

Comment:
The version in the heading just needs the `-canary.0` added to the tag.

Reply:
Haha damn it, so close :D